Packs/ANYRUN/Integrations/AnyRunSandbox/AnyRunSandbox.yml

```diff
@@ -0,0 +1,1476 @@
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.options.mitm          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.options.heavyEvasion          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.options.fakeNet          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.options.additionalTime          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.options.timeout          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.tags
+          type: Unknown
+          description: ''
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.stopExecText
+          type: Date
+          description: ''
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.stopExec
+          type: Date
+          description: ''
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.creationText
+          type: Date
+          description: ''
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.creation
+          type: Date
+          description: ''
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.duration          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.sandbox.plan.name          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.sandbox.name          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.reports.graph          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.reports.STIX          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.reports.HTML          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.reports.MISP          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.reports.IOC          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.permanentUrl          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.analysis.uuid          
+          description: ""
+          type: String
+        - contextPath: ANYRUN.SandboxAnalysis.status          
+          description: ""
+          type: String
+    - arguments:
+        - description: 'Sandbox task uuid.'
+          name: task_uuid
+          required: true
+          type: String
+      name: anyrun-download-analysis-pcap
+      description: 'Returns the analysis network traffic dump.'
+      outputs:
+        - contextPath: ANYRUN.SandboxAnalysisReportPcap
```

Comment:
are you sure this command have a context output
from the code it's like there is no context output under this context path